todolist/settings.py

```diff
@@ -23,7 +23,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS = ['0.0.0.0', 'localhost', '127.0.0.1']
```

Comment:
The `ALLOWED_HOSTS` setting currently includes only local addresses. For production, you should specify the actual domain names or IP addresses that your Django application will be accessible from.